template-package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "",
+  "version": "",
+  "description": "",
+  "repository": {},
+  "authors": [],
+  "license": "",
+  "bin": {},
+  "dependencies": {}
+}
```

Comment:
template file doesn't have a scripts section. Do you plan to add this in #16?

Reply:
Keep in mind that this is a first iteration on all this stuff, and it's just some default values. It will get built out further in future iterations. I'm not against stubbing out a scripts section, but that can also be added later when we actually have scripts generated.

Reply:
Only reason I asked this was because #16 generates the index file. So, I was wondering if we plan on putting it in there. The original issue talks about adding a scripts section.

Reply:
It should have the scripts section, with a start script. The dockerfile calls the start command.

Reply:
Only if Docker was chosen as an option. We should simply run the start script with the index file for now and make a new issue to address your comment.

Reply:
I will add the scripts section in issue # 16 if that's alright.  I'm going to do that right now